src/enums/icon.ts

```diff
@@ -0,0 +1,4 @@
+export enum IconType {
```

Comment:
I like that you think about stuff like this :+1: 
However, in this particular case I'm not sure how useful this file will be. We do not have a lot of icon names that are hard coded. The `EMAIL` one in here is not used so it should be removed, which just leaves the `IMPORTANCE` one left.
What do you think about using a constant inside `input-field.tsx` instead?

Reply:
In this case, you are right that we don't have to use the enum.  It is good to use enums for the future I guess.